docs/event-loop.md

```
@@ -0,0 +1,73 @@
+`Window.Clock().ProcessEventsWhile()` will wait as long as a specific condition
+is true.
+
+> [!WARNING]
+>
+> `ProcessEvents()`/`ProcessEventsWhile()` are new functions added as the
+> behaviour was necessary to write predictable test code when using `fetch`. The
+> entire scenario exposed a problem not properly anticipated originally, and as
+> a consequence, the entire clock/event loop may require some changes.
+
+
+[^1]: There are tuning parameters for this behaviour; but not exposed. Please
```

markdownlint-cli2 (0.17.2)

Link and image reference definitions should be needed
Unused link or image reference definition: "^1"

(MD053, link-image-reference-definitions)